code/controllers/subsystem/communications.dm

```diff
@@ -7,6 +7,7 @@ SUBSYSTEM_DEF(communications)
 
 	var/silicon_message_cooldown
 	var/nonsilicon_message_cooldown
+	var/emergency_meeting_cooldown
```

Comment:
cooldown defines

Reply:
it still doesnt use the cooldown defines

Reply:
do you mean the cooldown macros? i set it to use a define for the cooldown time